doc/dash/dash-sonic-hld.md

```diff
@@ -364,7 +366,7 @@ admin_state              = Enabled after all configurations are applied.
 vnet                     = Vnet that ENI belongs to
 pl_sip_encoding          = Private Link encoding for IPv6 SIP transpositions; Format "field_value/full_mask" where both field_value and `full_mask` must be given as IPv6 addresses. field_value must be used as a replacement to the
 			   first (128-len(full_mask)) bits of pl_sip. Last 32 bits are reserved for the IPv4 CA. Logic: ((pl_sip & !full_mask) | field_value).
-pl_underlay_sip          = Underlay SIP (ST GW VIP) to be used for all private link transformation for this ENI
+pl_underlay_sip          = Underlay SIP (ST GW VIP) to be used for all private link transformation for this ENI - (Obsoleted - Will use overlay transpositions from mapping tables)
```

Comment:
Are both `pl_sip_encoding` and `pl_underlay_sip` being deprecated here?

Reply:
Corrected. Only `pl_sip_encoding `is deprecated